data/json/furniture.json

```diff
@@ -993,6 +993,42 @@
         }
     },{
         "type" : "furniture",
+        "id" : "f_bigmirror",
+        "name": "standing mirror",
+        "symbol": "{",
+        "color": "white",
+        "move_cost_mod": 2,
+        "required_str": 5,
+        "flags": ["NOITEM","BLOCKSDOOR"],
+        "bash": {
+            "str_min": 5, "str_max": 40,
```

Comment:
Rolls a value between str_min and str_max. If it's lower than attack power, breaks the object. In case of bashing, attack power is weapon bashing dmg + user's strength.